worker/resolve_connections.go

```diff
@@ -4,7 +4,7 @@ import "github.com/Cretezy/dSock/common"
 
 func resolveConnections(options common.ResolveOptions) ([]*SockConnection, bool) {
 	if options.Connection != "" {
-		sockConnection, exists := connections[options.Connection]
+		sockConnection, exists := connections.Connections[options.Connection]
```

Comment:
accessing to a map concurrently is not safe

Reply:
Alright will switch for RWMutux, didn't know Go 1.6 had changed that.